### DISCUSSION TITLE
Stuck in a loop on verification

### DISCUSSION THREAD

**Discussion by bandanagyrl** [community/community#138310] [GitHub Education]
### Hi everyone,

Ok so I have enrolled in school, have both my enrollment letter and my schedule. I have submitted both to GitHub. 

They asked for my billing info to be updated and my profile. I did both. They asked for my billing again, did it and logged out/back in and resubmitted. And rejected again - my name isn’t complicated - it’s spelled the same on my enrollment letter and on the billing info I have submitted. And yet I get rejected and I don’t know how to fix this?

I see my school was also added since my first application and my email matches the school - why is this so hard?

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*
